python-all.md

```diff
@@ -0,0 +1,40 @@
+# Python all(iterable)
+
+`all()` is a built in function in Python 3, to check if all items of an [_iterable_](https://docs.python.org/3/glossary.html#term-iterable) is `True`. It takes one argument,  `iterable`. 
```

Comment:
"built in function in Python 3": Repetitive use of the word `in`.
Maybe change to: "built in function with Python 3".